src/observability/index.ts

```
@@ -0,0 +1,921 @@
+		 */
+		const runHealthCheck = (
+			checkName: string
+		): Effect.Effect<HealthCheck, never> =>
+			Effect.gen(function* () {
+				const startTime = Date.now()
+				
+				let result: HealthCheck
+				
+				switch (checkName) {
+					case 'memory':
+						const memInfo = process.memoryUsage()
```

Biome (2.1.2)

[error] Other switch clauses can erroneously access this declaration.
Wrap the declaration in a block to restrict its access to the switch clause.

The declaration is defined in this switch clause:

Safe fix: Wrap the declaration in a block.


(lint/correctness/noSwitchDeclarations)